tests/unit/components/LemmaInput.test.ts

```diff
@@ -0,0 +1,44 @@
+import { DOMWrapper, mount } from '@vue/test-utils';
+import LemmaInput from '@/components/LemmaInput.vue';
+
+describe( 'LemmaInput', () => {
+
+	function createComponent( config: Record<string, unknown> = {} ) {
+		return mount( LemmaInput, {
+			props: {
+				modelValue: '',
+			},
+			...config,
+		} );
+	}
+
+	function findInput( wrapper: ReturnType<typeof createComponent> ): Omit<DOMWrapper<HTMLInputElement>, 'exists'> {
+		return wrapper.get( 'input' );
+	}
+
+	describe( ':props', () => {
```

Comment:
I really like this testing structure, never seen it before, but definitely adopted!

Reply:
I glimpsed it in a talk about vue testing and really liked it a lot as well: https://www.youtube.com/watch?v=OIpfWTThrK8
No idea how found this talk, though. It is from way before I had ever looked at vue^^